code/modules/industrial_lift/elevator/elevator_panel.dm

```diff
@@ -64,6 +69,9 @@
 	// this is just here for redundancy's sake.
 	. = INITIALIZE_HINT_LATELOAD
 
+	if(elevator_music_toggle)
+		elevator_music = new(src, TRUE)
```

Comment:
```suggestion
		elevator_music = new(src, /* name_of_paremeter_here = */ TRUE)
```

Reply:
@Mothblocks what to do if I have no idea? Just from the SM code I got that it must be that way, for some reason

Reply:
I think it's like "Is it turned on or off?" as initialization of shower has it FALSE

Reply:
From what I see it doesn't really have a name so yeah

Reply:
Uh, it definitely has a name.

Look in `/datum/sound_loop/New` and see what the parameter is called, then copy that name.

Reply:
> Uh, it definitely has a name.
> 
> Look in `/datum/sound_loop/New` and see what the parameter is called, then copy that name.

Why every other can just have "TRUE" and "FALSE"? Whatever.
I will find it, if I won't then bleep

Reply:
They're not supposed to. It's hard to globally enforce code standards. We're working on it.

All you have to do is search `/datum/sound_loop/New` and figure out what the first parameter is.

Reply:
D o n e